docs/general-development/sharepoint-add-ins-compared-with-sharepoint-solutions.md

```diff
@@ -9,117 +9,66 @@ ms.localizationpriority: high
-  
-    
-    
+- New SharePoint Add-ins are self-contained extensions that may include cloud-based logic and data, SharePoint components, and client-side scripts, but not custom managed code that runs on SharePoint servers. They are installed from either the Office Store or an organization add-in catalog, and can be installed on either on-premises farms or Microsoft SharePoint Online. For an overview of SharePoint Add-ins, see  [SharePoint Add-ins](https://learn.microsoft.com/sharepoint/dev/sp-add-ins/sharepoint-add-ins).
+
+- SharePoint farm solutions are packages of SharePoint components that are uploaded to a farm-wide gallery from where they can be installed. They cannot be distributed through the Office Store, and they cannot be installed on SharePoint Online. They can include custom managed code that runs on the SharePoint farm servers. For more information about the basics of farm solutions, see  [Solutions Overview](https://learn.microsoft.com/previous-versions/office/developer/sharepoint-2010/aa543214(v=office.14)) and [Farm Solutions in SharePoint 2010](https://learn.microsoft.com/previous-versions/office/developer/sharepoint-2010/gg575564(v=office.14)).
 
+- NCSSs are also packages of SharePoint components; but they are uploaded to a site collection gallery from where they can be installed. They can be installed to either on-premises farms or to SharePoint Online, but they cannot be distributed through the Office Store. They can include the almost the same kinds of descriptive components as SharePoint Add-ins and, like add-ins, they can have JavaScript, but they do not contain custom managed code that runs on the SharePoint servers. Differences in the deployment systems of add-ins and NCSSs make NCSSs a better development option for a short list of scenarios. For information about sandboxed solutions, see  [Sandboxed Solutions in SharePoint 2010](https://learn.microsoft.com/previous-versions/office/sharepoint-server-2010/ee721992(v=office.14).
+
+> **Important:**
```

Comment:
Incorrect use of the important alert... please correct to use correct version. [See this as an example](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/apis/migration-api-overview.md?plain=1#L20)